apps/site/assets/ts/schedule/components/reducer.ts

```diff
@@ -82,37 +51,17 @@ export const iconOpts = (
 const zIndex = (icon: string | null): number | undefined =>
   icon === "vehicle-bordered-expanded" ? 1000 : undefined;
 
-const updateMarker = (marker: Marker): Marker => ({
+export const updateMarker = (marker: Marker): Marker => ({
   ...marker,
-  tooltip: (
-    <div>
-      {marker.vehicle_crowding && (
-        <>
-          <CrowdingPill crowding={marker.vehicle_crowding} />
-          <br />
-        </>
-      )}
-      {marker.tooltip_text}
-    </div>
-  ),
   icon_opts: iconOpts(marker.icon), // eslint-disable-line camelcase
   z_index: zIndex(marker.icon) // eslint-disable-line camelcase
 });
 
-const isVehicleMarker = (marker: Marker): boolean =>
-  marker.icon ? marker.icon.includes("vehicle") : false;
-
-interface IdHash {
-  [id: string]: true;
-}
-
 const shouldRemoveMarker = (id: string | null, idHash: IdHash): boolean =>
   id !== null && idHash[id] === true;
 
-interface State {
-  channel: string;
-  markers: Marker[];
-}
+const isVehicleMarker = (marker: Marker): boolean =>
+  marker.icon ? marker.icon.includes("vehicle") : false;
```

Comment:
These map-related functions feel kind of out of place here, could we move these to a helper file or something?

Reply:
Oops, this didn't highlight the functions I wanted -- I was talking about `iconOpts`, `zIndex`, `updateMarker`, `shouldRemoveMarker`, `isVehicleMarker`